### PR TITLE
structure changes to the error view

### LINKED ISSUE #189: Show that we are open source on our homepage
We need to show that we are opensource on our homepage and link to our github repo. This can be in the footer or as a section below all the information.

### PR DESCRIPTION
**Pull request by Panioglo**
Fixes #189, #190, #191, #193, #192, #196

### CODE REVIEW

**Comment by Panioglo** [comment]
open source
http://pasteboard.co/a5L53VF7n.png

**Comment by Panioglo** [comment]
slides
http://pasteboard.co/a5NMuNcli.png
http://pasteboard.co/a5N8ESNer.png
http://pasteboard.co/a5OhGmHVq.png
http://pasteboard.co/a5OP8g5pC.png

File: app/views/home/index.html.haml [hunk 1]
```diff
@@ -17,7 +17,7 @@
     .col-md-4
       %img{:src => asset_path("icon-2.png")}/
       %h3 Fast Solutions
-      %p Once we are aware of your issue, we quickly proceed to taking care of it
+      %p Once we are aware of yur issue, we quickly proceed to taking care of it
```
Comment (Chocksy) [inline app/views/home/index.html.haml]: you made a typo here...

File: app/views/home/index.html.haml [hunk 2]
```diff
@@ -34,23 +34,29 @@
...
+        %li{"data-slide-to" => "3", "data-target" => "#carousel"}
       / Wrapper for slides
       .carousel-inner{:role => "listbox"}
         .item.active
           .carousel-caption
-            %h4 Keep your users informed
-            %p Email your users or post a message on your website for them to see
-          %img{:src => asset_path("slides/1.jpg")}/
+            %h4 Keep track of your errors
+            %p Configure your website with the code we give you and we will take care of your errors
```
Comment (Chocksy) [inline app/views/home/index.html.haml]: Install the client code and we'll store all your application errors

File: app/views/home/index.html.haml [hunk 3]
```diff
@@ -34,23 +34,29 @@
...
+          %img{:src => asset_path("slides/img1.png")}/
```
Comment (Chocksy) [inline app/views/home/index.html.haml]: Also try to replace this image name with `exceptions_screenshot.png`

File: app/views/home/index.html.haml [hunk 4]
```diff
@@ -34,23 +34,29 @@
...
         .item
           .carousel-caption
-            %h4 Keep your users informed
-            %p Email your users or post a message on your website for them to see
-          %img{:src => asset_path("slides/1.jpg")}/
+            %h4 Stay informed
+            %p Go to an individual error to see more about what happened
+          %img{:src => asset_path("slides/img2.png")}/
```
Comment (Chocksy) [inline app/views/home/index.html.haml]: change image name to `individual_error.png`

File: app/views/home/index.html.haml [hunk 5]
```diff
@@ -34,23 +34,29 @@
...
         .item
           .carousel-caption
-            %h4 Keep your users informed
-            %p Email your users or post a message on your website for them to see
-          %img{:src => asset_path("slides/1.jpg")}/
+            %h4 Check the exception stacktrace
+            %p Click the expand button in the exception panel to see more details about the error
+          %img{:src => asset_path("slides/img3.png")}/
```
Comment (Chocksy) [inline app/views/home/index.html.haml]: change image name to `exception_details.png`

File: app/views/home/index.html.haml [hunk 6]
```diff
@@ -34,23 +34,29 @@
...
+        .item
+          .carousel-caption
+            %h4 Follow the breadcrumbs
+            %p See each crumb and follow the error timestamp appearance
```
Comment (Chocksy) [inline app/views/home/index.html.haml]: See the action that lead to the error with the breadcrumb feature.

File: app/views/home/index.html.haml [hunk 7]
```diff
@@ -34,23 +34,29 @@
...
+          %img{:src => asset_path("slides/img4.png")}/
```
Comment (Chocksy) [inline app/views/home/index.html.haml]: change name to `breadcrumb_screenshot.png`

**Comment by Panioglo** [comment]
http://pasteboard.co/ayFdLQHID.png

**Pull request merged**